versioned_docs/version-8.6/self-managed/concepts/elasticsearch-without-cluster-privileges.md

```diff
@@ -0,0 +1,282 @@
+---
+id: elasticsearch-without-cluster-privileges
+title: "Elasticsearch without cluster privileges"
+---
+
+In case the Camunda single application cannot access Elasticsearch with cluster privileges, it is possible to run the schema manager as a stand-alone application separate from the main application. The schema manager solely creates all necessary Elasticsearch Indices. In this case, elevated privileges are only required for the schema creation, the single application does not need to have cluster privileges to work any more.
+
+:::note Database Support
+This feature is only available from version 8.6.10 on and is also only supported for Elasticsearch installations (no OpenSearch support).
+:::
+
+:::note Essential privileges required by the single application
+An index-level privilege of at least `manage` is still required for the Camunda single application to work properly.
+:::
+
+## Setup
+
+For this setup to work, the database schema must be initialized first (step 1). This requires cluster level privileges for the database. This only needs to be executed once.
+Once the schema is initialized, the application can be started without cluster level privileges (step 2). The steps are described in detail below.
+
+### 1. Initialize the Schema Manager
+
+The schema manager is started as a separate standalone Java application and is responsible for creating and managing the database schema and applying database settings, such as retention policies for example.
+This step requires cluster level privileges for the database and only needs to be executed once per installation.
```

Comment:
:question: Do we need to specify the required permissions? Or is there an ES equivalent, i.e. saying `superuser` or something (is that a thing for ES?).